app/routes.py

```diff
@@ -1,2 +1,49 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    
+    def __init__(self, name, id, description, has_life=None):
+        self.name = name
+        self.id = id
+        self.description = description
+        self.has_life = has_life or False
+    
+    def to_json(self):
+        return {
+            "name": self.name,
+            "id": self.id,
+            "description": self.description,
+            "has_life": self.has_life
+        }
+
+planets = [
+    Planet("Mercury", 1, "small, grey-silver", False),
+    Planet("Venus", 2, "hot, bright", False),
+    Planet("Earth", 3, "beautiful", True)
+]
+
+planet_bp = Blueprint("planets", __name__, url_prefix= "/planets")
+
+def validate_id(id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message": f"{id} is not a valid planet id."},400))
+    for planet in planets:
+        if planet.id == id:
+            return planet
+    abort(make_response({"message": f"Planet with id of {id} was not found"},404))
+
+
+@planet_bp.route("", methods=["GET"])
+def get_all_planets():
```

Comment:
Nice job naming the method to describe what this route does. 

In the future, if you end up splitting your routes up into distinct route files (cat_routes.py and dog_routes.py for example) then you could have method names like get_all() and get_one() since we know that all the routes in cat_routes.py are related to the Cat class.